proparse/src/main/java/org/prorefactor/proparse/antlr4/PostLexer.java

```diff
@@ -197,13 +210,17 @@ private void preproElseif() throws IOException, TokenStreamException, Recognitio
   }
 
   private void preproEndif() throws IOException {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'java.io.IOException', as it cannot be thrown from method's body. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.riverside-software.fr/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)